integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLifecycleSampleScripts.java

```diff
@@ -0,0 +1,446 @@
+    createSecretForBaseImages(domainNamespace);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    //create and start WebLogic domain using domain-home-on-pv sample scripts
+    createDomain(sampleBase);
+  }
+
+  /**
+   * Test scripts for stopping and starting a managed server.
+   */
+  @Test
+  @MethodSource("paramProvider")
+  @DisplayName("Test server lifecycle samples scripts")
+  public void testServerLifecycleScripts() {
+
+    // Verify that stopServer script execution shuts down server pod and replica count is decremented
+    String serverName = managedServerNameBase + "1";
+    executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName);
+    checkPodDoesNotExist(managedServerPodNamePrefix + "1", domainName, domainNamespace);
+    assertDoesNotThrow(() -> {
+      checkClusterReplicaCountMatches(clusterName, domainName, domainNamespace, 1);
+    });
+
+    // Verify that startServer script execution starts server pod and replica count is incremented
+    executeLifecycleScript(START_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName);
+    checkPodExists(managedServerPodNamePrefix + "1", domainName, domainNamespace);
+    assertDoesNotThrow(() -> {
+      checkClusterReplicaCountMatches(clusterName, domainName, domainNamespace, 2);
+    });
+  }
+
+  /**
+   * Test scripts for stopping and starting a managed server while keeping replica count constant.
+   */
+  @Test
+  @MethodSource("paramProvider")
+  @DisplayName("Test server lifecycle samples scripts with constant replica count")
+  public void testServerLifecycleScriptsWithConstantReplicaCount() {
+    String serverName = managedServerNameBase + "1";
+    String keepReplicaCountConstantParameter = "-k";
+    // Verify that replica count is not changed when using "-k" parameter and a replacement server is started
+    executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName, keepReplicaCountConstantParameter);
+    checkPodDoesNotExist(managedServerPodNamePrefix + "1", domainName, domainNamespace);
+    checkPodExists(managedServerPodNamePrefix + "3", domainName, domainNamespace);
+    assertDoesNotThrow(() -> {
+      checkClusterReplicaCountMatches(clusterName, domainName, domainNamespace, 2);
+    });
+
+    // Verify that replica count is not changed when using "-k" parameter and replacement server is shutdown
+    executeLifecycleScript(START_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName, keepReplicaCountConstantParameter);
+    checkPodExists(managedServerPodNamePrefix + "1", domainName, domainNamespace);
+    checkPodDoesNotExist(managedServerPodNamePrefix + "3", domainName, domainNamespace);
+    assertDoesNotThrow(() -> {
+      checkClusterReplicaCountMatches(clusterName, domainName, domainNamespace, 2);
+    });
+  }
+
+  /**
+   * Test scripts for stopping and starting a cluster.
+   */
+  @Test
+  @MethodSource("paramProvider")
```

Comment:
Since the tests are not parameterized you can remove this line.

Reply:
Done.